src/main/java/randoop/reflection/ClassLiteralExtractor.java

```diff
@@ -21,22 +25,43 @@ class ClassLiteralExtractor extends DefaultClassVisitor {
   /** Map from a class under test to the literal sequences that appear in it. */
   private MultiMap<ClassOrInterfaceType, Sequence> literalMap;
 
+  /** The storage for constant information. */
+  private ScopeToConstantStatistics scopeToConstantStatistics;
+
   /**
    * Creates a visitor that adds discovered literals to the given map.
    *
-   * @param literalMap a map from types to sequences in them that yield a constant
+   * @param literalMap map from a type to sequences in it that yield a constant
    */
   ClassLiteralExtractor(MultiMap<ClassOrInterfaceType, Sequence> literalMap) {
     this.literalMap = literalMap;
+    this.scopeToConstantStatistics = new ScopeToConstantStatistics();
+  }
+
+  /**
+   * Creates a visitor that adds discovered literals to the given map and records constant
+   * statistics. Only used when constant-tfidf is enabled.
```

Comment:
I think all uses of constants should obtain them from the same source, so "Only used when constant-TF-IDF is enabled." is a bad specification, here and elsewhere.